components/admin_console/billing/billing_summary/billing_summary.tsx

```diff
@@ -10,19 +10,18 @@ import {BillingSchemes, CloudLinks, TrialPeriodDays} from 'utils/constants';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 import BlockableLink from 'components/admin_console/blockable_link';
 import OverlayTrigger from 'components/overlay_trigger';
-
-import noBillingHistoryGraphic from 'images/no_billing_history_graphic.svg';
-import upgradeMattermostCloudImage from 'images/upgrade-mattermost-cloud-image.svg';
+import UpgradeSvg from 'components/common/svg_images_components/upgrade.svg';
+import ResetPasswordSvg from 'components/common/svg_images_components/reset_password.svg';
 
 import {trackEvent} from 'actions/telemetry_actions';
 
 import {Client4} from 'mattermost-redux/client';
 
 export const noBillingHistory = (
     <div className='BillingSummary__noBillingHistory'>
-        <img
-            className='BillingSummary__noBillingHistory-graphic'
```

Comment:
Nit: can we remove the class definition in `components/admin_console/billing/billing_summary/billing_summary.scss`?